recipe/meta.yaml

```diff
@@ -88,32 +88,53 @@ outputs:
         - {{ cdt('pciutils-devel') }}        # [linux]
         - {{ cdt('libxscrnsaver-devel') }}   # [linux]
         - {{ cdt('libxtst-devel') }}         # [linux]
+        - {{ cdt('libselinux') }}            # [linux]
         - {{ cdt('libselinux-devel') }}      # [linux]
         - {{ cdt('libxdamage') }}            # [linux]
+        - {{ cdt('libxdamage-devel') }}      # [linux]
         - {{ cdt('libxfixes') }}             # [linux]
+        - {{ cdt('libxfixes-devel') }}       # [linux]
         - {{ cdt('libxxf86vm') }}            # [linux]
-        - {{ cdt('mesa-libgbm') }}           # [linux]
-        - {{ cdt('pcre') }}                  # [linux and cdt_name != 'cos6']
-        - make                               # [not win]
+        - {{ cdt('libxxf86vm-devel') }}      # [linux]
+        - {{ cdt('libsm-devel') }}           # [linux]
+        - {{ cdt('libice-devel') }}          # [linux]
+        - {{ cdt('libxcb') }}                # [linux]
+        - {{ cdt('libxcb-devel') }}          # [linux]
+        - {{ cdt('expat-devel') }}           # [linux]
+        - {{ cdt('pcre') }}                  # [linux]
+        - {{ cdt('libglvnd-glx') }}          # [linux]
+        - {{ cdt('libglvnd-egl') }}          # [linux]
+        - {{ cdt('libxkbcommon-devel') }}    # [linux]
+        - {{ cdt('xcb-util-devel') }}        # [linux]
+        - {{ cdt('xcb-util-image-devel') }}  # [linux]
+        - {{ cdt('xcb-util-keysyms-devel') }}     # [linux]
+        - {{ cdt('xcb-util-renderutil-devel') }}  # [linux]
+        - {{ cdt('xcb-util-wm-devel') }}          # [linux]
         - jom                                # [win]
-        - pkg-config                         # [not win]
+        - python                                 # [build_platform != target_platform]
+        - cross-python_{{ target_platform }}     # [build_platform != target_platform]
+        - pyqt-builder                           # [build_platform != target_platform]
+        - sip                                    # [build_platform != target_platform]
+        - make
       host:
-        - {{ pin_subpackage('pyqt5-sip', exact=True) }}
-        - {{ cdt('mesa-libgl-devel') }}      # [linux]
-        - alsa-lib                           # [linux]
-        - dbus                               # [not win]
-        # one unconstrained to pick up global pinning + a lower bound
-        - qt
-        - qt >=5.12.9
         - python
+        - pip
+        - sip                                    # [build_platform == target_platform]
+        - toml
+        - pyqt-builder                           # [build_platform == target_platform]
+        - packaging
+        - qt-main
```

Comment:
This is still somewhat problematic right now.

Yesterday, I thought I could build the meta package, `qt` that depended on both `qt-main` and `qt-webengine`. The problem is that 
* qt-main is version 5.15.2 
* qt-webengine is version 5.15.4

They both depend on the `qt` package with `x.x.x` as pin.

The issue is that qt `5.15.4` seems to be a commercial only release that doesn't have a download file available on the opensource download page. I'm not too sure how to relax this constraint. Perhaps we should relax the constraint to `x.x` and not `x.x.x`.

Reply:
So both [qt-main](https://github.com/conda-forge/qt-main-feedstock/blame/main/recipe/meta.yaml#L51) and [qt-webengine](https://github.com/conda-forge/qt-webengine-feedstock/blame/main/recipe/meta.yaml#L31)  already use the `x.x` pinning, so it seems to me that this should be possible...?

However, the [run-constraint](https://github.com/conda-forge/qt-main-feedstock/blame/main/recipe/meta.yaml#L175) of qt-main -> qt was added by @isuruf precisely in the context of the [split](https://github.com/conda-forge/qt-main-feedstock/commit/aec1b1044b7d4552c8c4022e3e1177df74aa2a17) into `qt-main`, so not sure if I'm missing something there.

Reply:
I guess you are missing the timeline of events.

I raised this concern, and worked through some of the issues to address it. Still need to migrate qt-webengine to the new icu